tsdb/head_wal.go

```diff
@@ -193,122 +194,127 @@ func (h *Head) loadWAL(r *wal.Reader, multiRef map[chunks.HeadSeriesRef]chunks.H
 	// The records are always replayed from the oldest to the newest.
 Outer:
 	for d := range decoded {
```

Comment:
The diff below is hard to read while I know nothing changed in the WAL replay. But to keep the indentation less for the large part, can you change it to this?
```go
Outer:
	for d := range decoded {
		select {
		case <-ctx.Done():
			break Outer
		default:
		}
		// ... everything else
	}
```

And `break Outer` because see this https://goplay.tools/snippet/J76Au0aZS3Q